Util/Schema/createTree.m

```diff
@@ -0,0 +1,74 @@
+function [stype, root, nrec] = createTree(arg1, stype, name, nrec, func),
+    % function [stype, root, nrec] = createTree(arg1, stype, name, nrec, func),
+    % A function to recursively apply a function handler @func,
+    % to all fields of an argument.
+    % the function generate the exact same type of arg1,
+    % visiting/applying @func through all nested structures (cells/structs)
+    %
+    % `arg1` = a matlab variable
+    % stype = the parent structure - internal use
+    % name = the name of this walk level - internal use
+    % nrec = the level of this walk level - internal use
+    % func = a special function handle that return 3 outputs - see DetectType
+    % <->
+    % stype = the result structure with follow-up nests/walks.
+    % root = is the result of @func for the root level of arg1.
+    % nrec = is the number of children visits.
+    %
+    % author: hugo.oliveira@utas.edu.au
```

Comment:
Can you please re-use the header template consistently across new scripts? This includes licencing info. Also, try to adopt the same indent style as with historical functions if possible.

Reply:
@ggalibert , I was supposed to say something about that but forgot. I don't think the license should be in every single m file. The license should be on the repository - all files within the repo should abide by the same license. This adds too much inflation to the codebase. I'm happy to add but it's just something I found useless

Reply:
I don't mind which way you want it to be done if you can confirm with the devs that the license doesn't need to be in every file header. However I think it's important to stick to a consistent template, whichever you think is best so there is probably a separate task of harmonising them at some stage to a new template, but in the meantime probably preferable to keep consistency with current template?

Reply:
just checked on GPL stuff - it is required on every file. I thought it was the same as MIT (just put a license file and all good). The copyright should also be in every file. 

PS: This kinda of information should be somewhere in a "contributing" section. Probably also worth to mention in the PR template (I will update the template that I made here).